### PR TITLE
Bug fix: Noise in classification script

### PR DESCRIPTION
**Pull request by simon-mauras**

### CODE REVIEW

**Comment by simon-mauras** [comment]
There is a problem in the getRank function. One easy way to have a perfect score is to predict always 0 for each class

File: assignments/ShapeIndexing/scriptClassification.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ NBIMGTESTS=40
 NBTESTS=5
 
 #No noise here
-MAXNOISE=0.0
+MAXNOISE=0.5
```
Comment (coma94) [inline assignments/ShapeIndexing/scriptClassification.sh]: I think that is normal to have no noise in the classification script ..?

Comment (simon-mauras) [inline assignments/ShapeIndexing/scriptClassification.sh]: In my last PR I updated the classification script, and as MAXNOISE was fixed to 0, I removed the noise :/ So it's a bug from me...

Comment (dcoeurjo) [inline assignments/ShapeIndexing/scriptClassification.sh]: well... let's assume that there is no noise in the classification test (remember, half of the image database is hidden :smile:). I'll edit your file.
Thanks for the getRank edit.

**Pull request merged**